### PR TITLE
More cleanup

### PR DESCRIPTION
**Pull request by liayn**

### CODE REVIEW

**Comment by liayn** [comment]
@marcoceppi Would you mind merging my long standing PR?

File: examples/server/setup.php [hunk 1]
```diff
@@ -22,16 +22,18 @@
  * Data.
  */
 
-$store_types = array("Filesystem" => "Auth_OpenID_FileStore",
+$store_types = [
+    "Filesystem" => "Auth_OpenID_FileStore",
```
Comment (marcoceppi) [inline examples/server/setup.php]: indent doesn't match

File: examples/server/setup.php [hunk 2]
```diff
@@ -22,16 +22,18 @@
...
                      "MySQL" => "Auth_OpenID_MySQLStore",
                      "PostgreSQL" => "Auth_OpenID_PostgreSQLStore",
-                     "SQLite" => "Auth_OpenID_SQLiteStore");
+                     "SQLite" => "Auth_OpenID_SQLiteStore"
```
Comment (marcoceppi) [inline examples/server/setup.php]: If we're going to do this, lets also use the dangling `,`

File: examples/server/lib/render/idpage.php [hunk 3]
```diff
@@ -18,14 +18,15 @@ function idpage_render($identity)
 {
     $xrdsurl = buildURL('userXrds')."?user=".urlencode($identity);
 
-    $headers = array(
-                     'X-XRDS-Location: '.$xrdsurl);
+    $headers = [
+                     'X-XRDS-Location: '.$xrdsurl
```
Comment (marcoceppi) [inline examples/server/lib/render/idpage.php]: Lets use a dangling `,`

File: examples/server/lib/common.php [hunk 4]
```diff
@@ -63,7 +63,8 @@ function doAuth($info, $trusted=null, $fail_cancels=false,
                            'postcode' => '12345',
                            'country' => 'ES',
                            'language' => 'eu',
-                           'timezone' => 'America/New_York');
+                           'timezone' => 'America/New_York'
```
Comment (marcoceppi) [inline examples/server/lib/common.php]: Dangling `,` here

File: examples/detect.php [hunk 5]
```diff
@@ -201,7 +201,8 @@ function detect_math($r, &$out)
             'security of the network links involved. If you are only ' .
             'using consumer support, you should still be able to operate ' .
             'securely when the users are communicating with a ' .
-            'well-implemented server.'));
+            'well-implemented server.'
```
Comment (marcoceppi) [inline examples/detect.php]: Dangling `,`

File: examples/consumer/common.php [hunk 6]
```diff
@@ -37,11 +37,11 @@ function doIncludes() {
 doIncludes();
 
 global $pape_policy_uris;
-$pape_policy_uris = array(
+$pape_policy_uris = [
 			  PAPE_AUTH_MULTI_FACTOR_PHYSICAL,
 			  PAPE_AUTH_MULTI_FACTOR,
 			  PAPE_AUTH_PHISHING_RESISTANT
```
Comment (marcoceppi) [inline examples/consumer/common.php]: trailing `,`

File: admin/texttest.php [hunk 7]
```diff
@@ -41,13 +41,15 @@ function microtime_float()
...
-$longopts = array('no-math',
+$longopts = [
+    'no-math',
                   'buggy-gmp',
                   'no-curl',
                   'math-lib=',
                   'insecure-rand',
                   'thorough',
-                  'extra-tests=');
+                  'extra-tests='
```
Comment (marcoceppi) [inline admin/texttest.php]: `,`

**Comment by liayn** [comment]
Wow... That was really fast. Will update the PR to match latest code (changes were done by PHPStorm back then)

**Pull request merged**